README.md

```diff
@@ -258,3 +258,19 @@ You also can use the provided `Dockerfile` to build your own image:
 ```shell
 docker build -t sgpt .
 ```
+
+## Development
+Install dependencies:
+```shell
+python setup.py install --user
+```
+We need to tell shell_gpt to import files from the local `sgpt/` folder, instead of from the installed `sgpt` package. So edit `app.py` and add these lines after `import os`:
```

Comment:
Then to install I just run `pip install -e .` to handle all the path stuff as well.

It is debatable if this is better though. It works well for installing a build as it would be installed from pip on a system, but makes it difficult to latch a debugger onto. 

For that I was running PyCharm and then executing app.py directly. PyCharm automatically adds the base repository to the path, thereby solving the issue of the sgpt module not being found.

Reply:
That's good to know!

Fun fact, if you try to do that when you already have `shell_gpt` installed, the error is completely unhelpful:

```
AttributeError: 'PathDistribution' object has no attribute 'key'
```

But `pip uninstall shell_gpt` followed by `pip install -e .` works fine.